Packs/Cyberint/Classifiers/classifier-Cyberint_(mapper).json

```diff
@@ -0,0 +1,590 @@
+{
+    "description": "",
+    "feed": false,
+    "id": "ac904a90-0668-4e4a-80f9-e62e3f18a1d3",
```

Comment:
Please change the ID here to match the mapper name, and then change the relevant RN entry - this should solve the validation errors.